plugins/cue_points/base/batch/CuePointBulkUploadXmlHandler.php

```diff
@@ -207,16 +207,17 @@ protected function parseCuePoint(SimpleXMLElement $scene)
 			$cuePoint->systemName = $scene['systemName'] . '';
 			
 		$cuePoint->startTime = kXml::timeToInteger($scene->sceneStartTime);
-	
-		$tags = array();
-		foreach ($scene->tags->children() as $tag)
+		if (isset($scene->tags))
 		{
-			$value = "$tag";
-			if($value)
-				$tags[] = $value;
+			$tags = array();
+			foreach ($scene->tags->children() as $tag) {
+				$value = "$tag";
+				if ($value)
+					$tags[] = $value;
+			}
+			$cuePoint->setTags(implode(',', $tags));
```

Comment:
$cuePoint is of type KalturaCuePoint and doesn't have setters, you should set the public tags property, or n other words, just revert this change.